bower.json

```diff
@@ -1,12 +1,11 @@
 {
   "name": "react-filterbar",
-  "version": "0.0.0",
```

Comment:
I'll throw up a PR adding it in, and suggest keeping it in line with the git tag. Bower seems to ignore the bower.json version key for the near future, at least until they get a proper package server.